src/fastmcp/server/tasks/elicitation.py

```
@@ -0,0 +1,224 @@
+                    "message": message,
+                    "requestedSchema": schema,
+                },
+            }
+        },
+    )
+
+    # Send notification (don't fail if this errors)
+    try:
+        await session.send_notification(notification)  # type: ignore[arg-type]
+    except Exception:
+        pass  # Best effort - task status is stored in Redis
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)